storybook/stories/chart/ComposedChart.stories.tsx

```diff
@@ -0,0 +1,132 @@
+import React from 'react';
+import { pageData } from '../data';
+import {
+  Area,
+  Bar,
+  Brush,
+  CartesianGrid,
+  ComposedChart as Composed,
+  Legend,
+  Line,
+  LineChart,
+  ReferenceDot,
+  ReferenceLine,
+  ResponsiveContainer,
+  Scatter,
+  Tooltip,
+  XAxis,
+  YAxis,
+} from '../../../src';
+
+export default {
+  title: 'Recharts/Chart/Composed Chart',
+  component: Composed,
+};
+
+// This render template can be shared across multiple stories.
+// All stories use the same data, but pass different children.
+const HorizontalChartTemplate = {
+  render: (args: any) => {
```

Comment:
I hate this `any` here but know why its needed 😅 

Could we maybe maybe this `Record<string, any>`? Just for a little bit of a stricter type

Reply:
It was late, and I had to submit for review :D I agree, and will iterate.